tox.ini

```diff
@@ -20,19 +28,16 @@ deps =
     xxhash
     crc32c
 commands =
-    py.test {posargs:--pylint --pylint-rcfile=pylint.rc --pylint-error-types=EF --cov=kafka --cov-config=.covrc}
+    pytest {posargs:--pylint --pylint-rcfile=pylint.rc --pylint-error-types=EF --cov=kafka --cov-config=.covrc}
 setenv =
     CRC32C_SW_MODE = auto
     PROJECT_ROOT = {toxinidir}
 passenv = KAFKA_VERSION
 
-[testenv:py26]
-# pylint doesn't support python2.6
-commands = py.test {posargs:--cov=kafka --cov-config=.covrc}
 
 [testenv:pypy]
 # pylint is super slow on pypy...
-commands = py.test {posargs:--cov=kafka --cov-config=.covrc}
+commands = pytest {posargs:--cov=kafka --cov-config=.covrc}
```

Comment:
Calls to `py.test` are deprecated in favor of `pytest`, https://stackoverflow.com/a/41893170/9852671 provides a useful explanation.